src/test.rs

```diff
@@ -439,6 +461,7 @@ pub async fn ca_contains_resources(handle: &Handle, resources: &ResourceSet) ->
         if ca_current_resources(handle).await.contains(resources) {
             return true;
         }
+        refresh_all().await;
```

Comment:
Was something broken here before or is this now needed because of the extension to the tests regarding suspension and unsuspension?